src/main/java/org/jala/efeeder/api/utils/ReadFileUtil.java

```diff
@@ -0,0 +1,46 @@
+package org.jala.efeeder.api.utils;
+
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.apache.commons.fileupload.FileItem;
+
+import com.opencsv.CSVReader;
+
+public class ReadFileUtil {
+
+	public static Map<String, List<String>> readCSVFile(FileItem fileItem) {
+
+		Map<String, List<String>> parameters = new HashMap<>();
+
+		CSVReader reader = null;
+
+		try {

```

Comment:
if it's possible use try-with-resources statement in order to avoid the finally block
https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html